run-tests.sh

```diff
@@ -1,10 +1,7 @@
 export CGO_ENABLED=0
 
-export USER=test
-export HOME=/home/test
-mkdir $HOME
-
-export PATH=$PATH:$GOPATH/bin
+export USER=${USER:-test}
```

Comment:
Default user is `test` but that user will not have a home dir. Since it's just alpine, why not default to `root`?

Reply:
Good idea. Will change default user to root.